app/src/main/java/org/ole/planet/myplanet/ui/myhealth/AddExaminationActivity.java

```diff
@@ -117,15 +117,15 @@ private void initExamination() {
             etVision.setText(examination.getVision());
             etHearing.setText(examination.getHearing());
             JsonObject encrypted = examination.getEncryptedDataAsJson(this.user);
-            etObservation.setText(JsonUtils.getString("notes", encrypted));
-            etDiag.setText(JsonUtils.getString("diagnosis", encrypted));
-            etTretments.setText(JsonUtils.getString("treatments", encrypted));
-            etMedications.setText(JsonUtils.getString("medications", encrypted));
-            etImmunization.setText(JsonUtils.getString("immunizations", encrypted));
-            etAllergies.setText(JsonUtils.getString("allergies", encrypted));
-            etXray.setText(JsonUtils.getString("xrays", encrypted));
-            etLabtest.setText(JsonUtils.getString("tests", encrypted));
-            etReferrals.setText(JsonUtils.getString("referrals", encrypted));
+            etObservation.setText(JsonUtils.getString(getString(R.string.note_), encrypted));
```

Comment:
Such strings have to be tagged **translatable="false"** because such strings are reference keywords from the database

Reply:
Where would I tag them?

Reply:
in the main string XML file